### PR TITLE
SDK Update - Customer Service

### PR DESCRIPTION
**Pull request by imvanzen**
Added following methods:
- Create Customer
- Update Customer
- Fetch Customer
- Delete Customer

All documented, verified

### CODE REVIEW

File: Customer.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    /**
+     * @param string $customerId
+     *
+     * Get customer details
+     *
+     * @throws Voucherify\ClientException
+     */
+    public function fetch($customerId) {
```
Comment (tpindel) [inline Customer.php]: @imvanzen  please change method name. must be consistent with other SDKs. Usse name `get`

**Comment by mandraszyk** [comment]
It's OK 👍

**Comment by mandraszyk** [comment]
Can we unify the namespaces declaration ? use `namespace xx;` or `namespace xx {}` in all files

**Comment by imvanzen** [comment]
@mandraszyk Yes we can. Moreover, since we have one class per file and only one defined `namespace` we should use `namespace xx;` convention. I started already with new implementation, but rest of classes have to wait for next iteration.

**Comment by mandraszyk** [comment]
@imvanzen: OK
We should include new namespace in readme: `use Voucherify\CustomerBuilder;`. Code is OK.

**Pull request merged**